src/dotnetTestExplorer.ts

```diff
@@ -141,12 +208,30 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
         });
     }
 
+    /**
+     * @description
+     * Checks to see if the @see{vscode.workspace.rootPath} is
+     * the same as the directory given, and resolves the correct
+     * string to it if not.
+     * @param dir
+     * The directory specified in the options.
+     */
+    private resolvePath(dir: string): string {
+        return path.isAbsolute(dir)
+            ? dir
+            : path.resolve(vscode.workspace.rootPath, dir !== vscode.workspace.rootPath ? dir : "");

```

Comment:
`dir !== vscode.workspace.rootPath ? dir : ""` Do we need it? `vscode.workspace.rootPath` would always be absolute